elastic-search-support/src/main/java/uk/gov/hmcts/ccd/definition/store/elastic/endpoint/ReindexTaskController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.ccd.definition.store.elastic.endpoint;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.ccd.definition.store.elastic.service.ReindexTaskService;
+import uk.gov.hmcts.ccd.definition.store.repository.model.ReindexTask;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(ReindexTaskController.REINDEX_TASKS_URI)
+@Api(value = ReindexTaskController.REINDEX_TASKS_URI)
+public class ReindexTaskController {
+
+    public static final String REINDEX_TASKS_URI = "/elastic-support/reindex/tasks";
```

Comment:
.authorizeHttpRequests(ar -> 
                ar.requestMatchers(ImportController.URI_IMPORT, ElasticsearchIndexController.ELASTIC_INDEX_URI)
                .hasAuthority("ccd-import")
                .anyRequest()
                .authenticated()
            )
            
            Add this end point in the SecurityConfiguration in above code